src/mlpack/tests/CMakeLists.txt

```diff
@@ -137,7 +137,7 @@ add_executable(mlpack_test
   main_tests/random_forest_test.cpp
   main_tests/softmax_regression_test.cpp
   main_tests/sparse_coding_test.cpp
-)
+  main_tests/kmeans_test.cpp)
```

Comment:
I think it's fine to leave the closing `)` on a separate line; it makes it just a little easier to add more files or move around their ordering.  I know that is a picky comment, sorry about that.